tensorflow_blade/workspace0.bzl

```diff
@@ -50,6 +51,48 @@ def _tf_blade_repositories():
         register_preinstalled_tools = True,  # just use the pre-installed.
     )
 
+    maybe_http_archive(
+        name = "rules_java",
+        sha256 = "f5a3e477e579231fca27bf202bb0e8fbe4fc6339d63b38ccb87c2760b533d1c3",
+        strip_prefix = "rules_java-981f06c3d2bd10225e85209904090eb7b5fb26bd",
+        urls = ["http://pai-blade.oss-cn-zhangjiakou.aliyuncs.com/bazelbuild/rules_java/archive/981f06c3d2bd10225e85209904090eb7b5fb26bd.tar.gz"],
```

Comment:
what requires java ?

Reply:
This is tmp for build on ecs-arm servers, cannot download `rule_java` from github urls.